tests/spec/phantesta-spec.js

```diff
@@ -98,6 +97,29 @@ describe('phantesta', function() {
         await phantesta.expectDiff('colorlighter', 'colorlight');
         await phantesta.expectDiff('colorlighter', 'colorgrey');
       }), 20000);
+      it('should be able to diff between different sizes', syncify(async function() {
+        var url1 = htmlServer.getUrl('/html/size.html?size=500x500');
+        var url2 = htmlServer.getUrl('/html/size.html?size=500x490');
+        await page.open(url1);
+        await phantesta.expect(page).toNotMatchScreenshot('bigger');
+        await phantesta.acceptDiff('bigger');
+        await page.open(url2);
+        await phantesta.expect(page).toNotMatchScreenshot('smaller');
+        await phantesta.acceptDiff('smaller');
+        await phantesta.expectDiff('bigger', 'smaller');
+      }), 20000);
+      it('should produce diff image for different sizes', syncify(async function() {
+        phantesta.startServer({host: 'localhost', port: '7993'});
```

Comment:
This doesn't need to take 20 seconds. At the end of the previous test, just check if the file at phantesta.getDiffPath exists